inc/inventory.class.php

```diff
@@ -322,6 +322,7 @@ function computersByOS($config = []) {
       $sql_states   = self::getStateCondition('c.states_id');
       $oses = ['Windows' => 'Windows',
                'Linux'   => 'Linux|Ubuntu',
+               'openSUSE'  => 'openSUSE',
```

Comment:
seems to me this should be `'Linux' => 'Linux|Ubuntu|openSUSE',` instead.

Reply:
yes, your appreciation is correct, I fixed this line